backend/api/tests/test_views.py

```diff
@@ -29,12 +29,17 @@ def setupOAuth(self):
         skip_authorization=True,
     )
     self.application.save()
+    scope = "read write"
+    for pk in stores:
+        scope += " store_" + str(pk) + ":vendor"
+        scope += " store_" + str(pk) + ":manager"
+        scope += " store_" + str(pk) + ":employee"
```

Comment:
@MattKersey Just to confirm. For these test cases, we are giving the `test-user` all scopes (vendor, manager, and employee) of permissions only for the stores we defined above, right? 

If that's the case, should we also include some test cases where someone without the proper scopes tries to do an out-of-scope request?